### ISSUE TITLE
Get quote for mussel genome bi-sulfide sequencing

### ISSUE THREAD

**Issue by sr320** [RobertsLab/resources#1884]

**Comment by kubu4** [comment]
Quote requests submitted to Psomagen and Zymo last Friday.

**Comment by kubu4** [comment]
Azenta quote requested today.

**Comment by kubu4** [comment]
Azenta quote:

`$16,830.00` (~$700/sample)

[30-1029194829_R1.pdf](https://github.com/RobertsLab/resources/files/15226641/30-1029194829_R1.pdf)

**Comment by kubu4** [comment]
Psoagen quote:

`$9,744` (~$400/sample)

[Quotation_ANQ00004754_20240506.pdf](https://github.com/RobertsLab/resources/files/15226789/Quotation_ANQ00004754_20240506.pdf)

**Comment by sr320** [comment]
@ChrisMantegna - what's the value of your award?

**Comment by ChrisMantegna** [comment]
5k

**Comment by kubu4** [comment]
Zymo quote: 

`$19,800` (~$825/sample)

[Quote_QUO17213_1715116487006.pdf](https://github.com/RobertsLab/resources/files/15241094/Quote_QUO17213_1715116487006.pdf)

**Comment by ChrisMantegna** [comment]
Are these numbers real?!?!? Geez. I knew this was pricey, but that's insane.

**Comment by sr320** [comment]
@kubu4 can you also get quotes for Illumina, PacBio, and nanopore through respective UW cores?

**Comment by kubu4** [comment]
Definitely. 

Reminder: UW core for Illumina sequencing will require us to make libraries ourselves. They do not offer bisulfite library prep.

**Comment by kubu4** [comment]
UW PacBio: `$27,149.34`

UW NanoPore:

- 24 flowcells: `$38,367.60`
  - [Estimate_NSC0158.pdf](https://github.com/RobertsLab/resources/files/15326906/Estimate_NSC0158.pdf)

- 12 flowcells: `23,901.00`
  - [Estimate_NSC0158(2).pdf](https://github.com/RobertsLab/resources/files/15326907/Estimate_NSC0158.2.pdf)

They've indicated they expect higher quality and higher yields for the 24 flowcells. The 12 flowcell procedure would run the samples through each flowcell two times, which reduces yield/quality.

**Comment by kubu4** [comment]
UW Northwest Genomics Center (sequencing only - libraries would have to be made ourselves):

- `$12,727.00`

[2024-05_012.pdf](https://github.com/RobertsLab/resources/files/15355724/2024-05_012.pdf)

**Comment by kubu4** [comment]
> Are these numbers real?!?!? Geez. I knew this was pricey, but that's insane.

I think the issue here is that the mussel genome is fairly large. It's at least 2x the size of the Pacific oyster genome, which is what we're usually dealing with.

**Comment by ChrisMantegna** [comment]
Hey All!

I want to check in on what we need to do to close the loop here and place the order before the end of the day on Friday so I don't forfeit ~4400 bucks. Did we choose a vendor?

**Comment by ChrisMantegna** [comment]
Which vendor?

**Comment by sr320** [comment]
Psomagen

**Comment by ChrisMantegna** [comment]
what budget number is attached to GR031727

**Comment by kubu4** [comment]
> what budget number is attached to GR031727

66-9283

**Comment by ChrisMantegna** [comment]
Placed the purchase order request via treq. thank you.

**Comment by ChrisMantegna** [comment]
I need work tags for the treq request - I don't know what those are
<img width="1296" alt="Screenshot 2024-05-31 at 9 20 26 AM" src="https://github.com/RobertsLab/resources/assets/81385272/22705bc3-8c26-490c-8ff1-381bc3d2bd49">

**Comment by sr320** [comment]
link to treq order?

**Comment by ChrisMantegna** [comment]
https://treq.environment.uw.edu/order/5403

**Comment by sr320** [comment]
Worktags for

GR031727: | 66-9283 | COD THERMAL STRESS | CC100950 CoEnv | Aquatic and Fishery Sciences (SAFS) - R&E |  | GR031727 | FD200 | RS100185 | FN110 |
GR006094: | 61-0552 | NSF-ROL | CC100950 CoEnv | Aquatic and Fishery Sciences (SAFS) - R&E | PG107267 | GR006094 | FD200 | RS100077 | FN110 |


you have to ask Sam S. for your award but she should have provided those #s

**Comment by ChrisMantegna** [comment]
OK - thank you. 
Jonas did the work tags last time, so I didn't even think to add them

**Comment by sr320** [comment]
I am pretty sure treq populates them for you.. I just cannot get in deep in your request to check.

**Comment by ChrisMantegna** [comment]
Disregard - I re-entered the budgets with the worktags. It will probably come back to you for approval some time today.

**Comment by ChrisMantegna** [comment]
The Psomagen guide for sample preparation is in the shiny new [repo](https://github.com/ChrisMantegna/WDFWmussels_epigenetics/blob/main/Psomagen%20general%20Sample%20prep%20instructions.pdf), but I need to clarify a few things before the samples can go out.

- The purchase request in TREQ has been approved but there is no purchase order number listed. I will followup in the morning since the status is "Place Order".

- What steps do I need to take to prepare the samples for shipment? 
-- Is there a labeling convention beyond sample ID, date, initials?
-- The guide recommends samples in TE buffer, is this common or do we have another standard practice?
-- Do I send all ~48ul of the extracted DNA?
-- What paperwork should go with the samples? I'm assuming random DNA showing up isn't standard protocol.

I'd like to do as much as possible before I go, but don't want to mess up any existing processes.

**Comment by ChrisMantegna** [comment]
Hello! 

I've sent the second request for the purchase order number and copied Jonas on it this time. Hopefully we will see movement.

Additionally, the sample concentrations are in the repo in the last message. Let me know if there is anything else you need and I'll let you know as soon as the PO is issued.

**Comment by kubu4** [comment]
You'll need to follow the guidelines the vendor suggests.

They should supply a sample submission sheet, as well as minimum sample quantity/concentration/volume.

If they don't you'll need to email them.

Regarding naming, you name the samples however you decide to name. They're already labelled, correct? Use the same names in the sample submission sheet.

---

> so I assumed the samples were sent out.

Ha! I assumed the same thing. I figured you'd received the PO number already, as I thought that's what we were waiting on.

**Comment by sr320** [comment]
So....   This issue is about getting a quote..... a quote was obtained. @kubu4 got the W "close" for resolving the issue.. 

This other conversation is not about a quote.  If there is another issue / to-do about getting samples sent then it needs to be another issue. 😃
